IDEAS/Buildings/Components/Examples/WallUnitTest.mo

```diff
@@ -98,4 +101,4 @@ First implementation.
 </li>
 </ul>
 </html>"));
-end WallUnitTest;
+end WallUnitTest;
```

Comment:
Can you add a newline at the end of the file? Furthermore, can you update the revision section.